tests/smoke/features/step_definitions/session.ts

```diff
@@ -10,9 +10,25 @@ Given('{string} has logged in', async function (this: World, stepUser: string):
   await loginPage.login({ user })
 })
 
+When('{string} logs in', async function (this: World, stepUser: string): Promise<void> {
```

Comment:
I'm not a huge fan of duplicating two full identical step definitions for the sake of them having a different notion of present/past ("logs in" vs "has logged in") since that will become painful when updating stuff...

Reply:
I will refactor to merge these two into a single function that has both variations of the step text.
(I forgot that I did a quick copy-paste to get it running, thanks for mentioning this!)

Reply:
Thanks!

Reply:
Can't see the changes yet, though

Reply:
Sadly I can't use the `When` keyword in a step form defined by a `Given` function. I will have to have both functions, and make them both call a common function to do the actual code.

Reply:
I sorted it out by making real functions rather than anonymous functions, and calling those from multiple places.